server/src/assets/entities/asset.entity.ts

```diff
@@ -32,6 +32,10 @@ export class Asset {
   @Column()
   quantity: number;
 
+  @Column('text', { array: true, default: '{}' })
```

Comment:
is this `default: '{}'` really how we set the default to an empty array?

Reply:
I had seen examples of it here:
https://stackoverflow.com/questions/57611633/typeorm-array-is-not-supported-in-postgres

But I think that empty object will return undefined unless it includes something in it. 

However, I changed it to:

`@Column('text', { array: true, default:["0, 0, 0"] })
    imgUrls: string[];`

This seems to work. 


I guess you can also return an empty array using hooks in typeORM?https://www.darraghoriordan.com/2021/10/19/populating-empty-arrays-type-orm/